bundles/org.openhab.binding.webthing/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="webthing"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Sample Thing Type -->
+	<thing-type id="generic">
+		<label>WebThing Binding Thing</label>
+		<description>Sample thing for WebThing Binding</description>
```

Comment:
The word "Binding" is redundant and can therefore be removed. "Sample" because the description is overwritten dynamically?

Reply:
fixed.